.github/workflows/patch.yml

```diff
@@ -0,0 +1,19 @@
+name: patch
+on:
+ pull_request:
+   branches:
+     - release/**
+   types:
+     - closed
+jobs:
+ main:
+   runs-on: ubuntu-latest
+   if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'patch/')
+   steps:
+     - uses: actions/checkout@v2
+     - run: git fetch origin ${{ github.event.pull_request.head.ref }}
+     - run: git checkout ${{ github.event.pull_request.head.ref }}
+     - run: git pull origin main
+     - run: git tag `python -c "import pyvista as pv; print(pv.__version__)"`
+     - run: git push origin `python -c "import pyvista as pv; print(pv.__version__)"`
+
```

Comment:
I think we need to configure git so the commit has an author or the action token is used somehow

Reply:
From 2019, but probably still the process: https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action/58393457#58393457

Reply:
Otherwise, I have used https://github.com/EndBug/add-and-commit

Reply:
Ref https://github.com/pyvista/pyvista/pull/2416/commits/3bc91c615b91ea584abcd8545c4300467db0580f.